src/Views/FirmwareView.vala

```diff
@@ -46,8 +46,8 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
         progress_alert_view.get_style_context ().remove_class (Gtk.STYLE_CLASS_VIEW);
 
         placeholder_alert_view = new Granite.Widgets.AlertView (
-            _("Checking for Updates"),
-            _("Connecting to the firmware service and searching for updates."),
```

Comment:
There is no ending ellipsis here. "The firmware service" is needless complexion telling the user nothing of value.
Explaining the connection made to it makes no sense, because it is just a technical aspect without real distinction.
What are "updates"? We deduce it has something to do with "firmware", which is not explained.